dependencies.yaml

```
@@ -0,0 +1,84 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.5"]
+      arch: [x86_64]
+    includes:
+      - build
+      - cudatoolkit
+      - develop
+      - doc
+      - py_version
+      - test_python
+  test_python:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - test_python
+  test_cpp:
+    output: none
+    includes:
+      - cudatoolkit
+      - test_cpp 
+  checks:
+    output: none
+    includes:
+      - build
+      - develop
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - conda-forge
+  - nvidia
+dependencies:
+  build:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cudf=22.12.*
+          - cmake>=3.23.1
```

```suggestion
          - cmake>=3.23.1,!=3.25.0
```
To sync with: https://github.com/rapidsai/cuspatial/pull/805/files